CONTRIBUTING.md

```diff
@@ -1,32 +1,32 @@
 # Contributing
 
-When contributing to this repository, please read through and get familiarised with the processes of making a change to this package.
+Please read through and get familiarised with the processes for introducing a
+change in Castor.
```

Comment:
Why sentences are now split into lines? IMO having one single line is better for different screen sizes.